src/inverseRobot.js

```diff
@@ -7,7 +7,30 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+  let count = 0;
+
+  for (const key of keys) {
```

Comment:
We can use robot object directly at this loop 
```suggestion
  for (const key in robot) {
```